spec/index.bs

```diff
@@ -2254,7 +2254,8 @@ handling should allow it to recover.
 Any {{GPUAdapter}} returned by {{GPU/requestAdapter()}} must provide the following guarantees:
 
 - At least one of the following must be true:
-    - {{GPUFeatureName/"texture-compression-bc"}} is supported.
+    - Both {{GPUFeatureName/"texture-compression-bc"}} and
+        {{GPUFeatureName/"texture-compression-bc-sliced-3d"}} are supported.
```

Comment:
Sorry, realized one really niche corner case is not covered by these rules: it would allow devices which have etc2+astc to have bc without bc-sliced-3d or vice versa. Added a rule for this below.

That change also makes this redundant, so to make the "BC or ETC2+ASTC" rule more symmetric I prefer to change the wording back:

```suggestion
- At least one of the following must be true:
    - {{GPUFeatureName/"texture-compression-bc"}} is supported.
```

Reply:
Previously committed suggestion